tests/requirements.txt

```diff
@@ -34,4 +33,4 @@ importlib-metadata==4.13.0
 # If this repository uses the setup functionality (e.g., script entry points)
 # of its own setup.py file, then this list must include an entry for that
 # setup.py file, e.g., '-e .' or '-e ./python' (without the quotes).
--e ./python
+-e ./../test_support
```

Comment:
See the comment on the Jenkinsfile - suggest making this relative to the tests directory.